src/parser/monk/mistweaver/modules/talents/InvokeChiJi.tsx

```diff
@@ -43,12 +155,25 @@ class InvokeChiJi extends Analyzer {
                     <li>{formatNumber(this.gustHealing)} healing from Chi-Ji Gust of Mist.</li>
                     <li>{formatNumber(this.envelopHealing)} healing from Enveloping Breath.</li>
                   </ul>
+                  Stack Breakdown:
+                    <ul>
+                    <li>{formatNumber(this.freeCasts)} free Enveloping Mist cast(s).</li>
+                    <li>{formatNumber(this.castsBelowMaxStacks)} Enveloping Mist cast(s) below max ({this.MAX_STACKS}) Chi-Ji stacks.</li>
+                    <li>{formatNumber(this.wastedStacks)} stack(s) wasted from overcapping Chi-Ji stacks.</li>
+                    </ul>
+                  Activity:
+                    <ul>
+                    <li>{(this.chijiGlobals/this.chijiUses).toFixed(2)} average gcds inside Chi-Ji window</li>
+                    </ul>
             </Trans>
```

Comment:
delete trans. Idc what other say its breaking things right now on production. We can loop back to this at a later date

Reply:
No!